packages/now-cli/src/util/load-doc.ts

```diff
@@ -0,0 +1,45 @@
+import fetch from 'node-fetch';
+import { Output } from './output';
+import chalk from 'chalk';
+import AbortController from 'abort-controller';
+
+export function loadDoc(
+  output: Output,
+  path: string
+): { print: (fallback?: string) => void } {
+  let md: string | null = null;
+
+  const controller = new AbortController();
+
+  fetch(`https://api-frameworks.zeit.sh/docs${path}`, {
+    signal: controller.signal,
+  })
+    .then(res => {
+      if (!res.ok) throw new Error();
+      return res.text();
+    })
+    .then(text => {
+      md = text;
+    })
+    .catch(() => {
+      output.debug(`Could not load doc ${path}, skipping`);
+    });
+
+  return {
+    print: fallback => {
+      // abort current request
+      controller.abort();
```

Comment:
I don't understand the purpose of abort() here. Why would you abort the current request?

Reply:
We abort the request once we print the message because we don't want to keep the "process" open. When calling print, we either:
- print the fetched doc if it is ready
- print the fallback if the doc is not ready
- don't print anything if the fallback is not defined and the doc is not ready

The idea is to make the `print` call always instant, as if it was a simple `console.log`.